### ISSUE TITLE
Doesn't play nice with redux-actions

### ISSUE THREAD

**Issue by baszalmstra** [samiskin/redux-electron-store#34]
We use redux-actions which has this defined when handling actions:
```
return (state = defaultState, action) => {
    if (!includes(actionTypes, action.type.toString())) {
      return state;
    }

    return (action.error === true ? throwReducer : nextReducer)(state, action);
  };
```
This is a problem for redux-electron-store when you do not pass in an initialState because of this line in `electronRendererEnhancer.js`:
```
let newInitialState = objectMerge(initialState || reducer(undefined, { type: null }), preload);
```
The type is set to null which causes an error in redux-actions.

**Comment by samiskin** [comment]
Hi, thanks for submitting the issue.  To confirm, would `@@redux/INIT` be the correct default `type` value here?  My intention with that line was to create an "initial state" by passing some blank action that would make everything give defaults.

**Comment by baszalmstra** [comment]
yeah, that sounds good.

**Comment by samiskin** [comment]
Should be fixed in `v0.4.1`
